origin/src/raw_mutex.rs

```diff
@@ -1,4 +1,8 @@
+use lock_api::{
+    GuardSend, Mutex as GenericMutex, MutexGuard as GenericMutexGuard, RawMutex as MutexApi,
```

Comment:
It'd clarify this code to rename the `RawMutex` struct here to something like `FutexMutex` or so, so that we don't need an `as` on the `RawMutex` from `lock_api`.